### DISCUSSION TITLE
Which support service is best for you?

### DISCUSSION THREAD

**Discussion by mahmoudajawad** [NvChad/NvChad#1733] [General]
In this links you've listed. Which one gets you most of the money your supporters send you? Also, a suggestion to enlist the one as preferred in the related section.

**Comment by siduck** [comment]
@mahmoudajawad as-salamu alaikum brother,  kofi/patreon are linked with my paypal account and paypal takes some transactional fees + fee of converting the USD to my currency.  

You can click on the sponsor button which is linked with my stripe account, they dont take any transactional fee, and you can pay with any method too, if you just have paypal then you can send with it

**Reply by mahmoudajawad** [reply]
W Alykum Alsalaam.
Got you.
